### DISCUSSION TITLE
Use a default locale that is different from page file names

### DISCUSSION THREAD

**Discussion by arnoson** [nuxt-modules/i18n#2557] [Q&A]
I have an app which has its default locale set to `de`.
But I want to code everything in english, therefore the pages are named like `pages/offers.vue`
Inside the page i define the `de` path
```js
// pages/offers.vue
<script setup>
  defineI18nRoute({ paths: { de: '/angebote' } })
</script>
```
I would expect that I can now use my file name to get the translated routes:
```js
localePath('/offers') // should return `/angebote`
```
Which doesn't work, it still returns `/offers`. I guess because `de` is my default locale it takes the filename as the `de` route and doesn't take into account my `defineI18nRoute`. Has anyone a similar setup and found a solution for when your default locale is not matching your page file names?

My config is:

```js
// nuxt.config.js
export default defineNuxtConfig({
  // ...
  i18n: {
    defaultLocale: 'de',
    locales: ['de'],
    strategy: 'prefix_except_default',
  },  
})
```

```js
// i18n.config.js
export default defineI18nConfig(() => ({
  legacy: false,
  locale: 'de',
}))
```

**Comment by lewebsimple** [comment]
I have the exact same issue and I had to disable the custom routes until a solution is found !
